autopush/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '1.2.3'
+__version__ = '1.3.0dev'
```

Comment:
This was mainly just to reflect that its not tagged yet. We could just use 1.3.0 I suppose even though its not tagged for it yet.